src/Widget.elm

```diff
@@ -543,3 +555,28 @@ tab :
     -> Element msg
 tab =
     Tab.tab
+
+
+
+{----------------------------------------------------------
+- PROGRESS INDICATOR
+----------------------------------------------------------}
+
+{-| Progress Indicator widget type
+
+If `maybeProgressPercent` is set to `Nothing`, an indeterminate progress indicator (e.g. spinner) will display.
+If `maybeProgressPercent` is set to `Just Int` (where the `Int` is completion percentage between 0 and 100 inclusive), a determinate progress indicator will visualize the completion percentage.
+
+-}
+type alias ProgressIndicator =
+    { maybeProgressPercent : Maybe Int }
```

Comment:
I'm not sure about using a singleton record. Additionally, I had expected a Float ranging between 0 and 1.

I feel like `Maybe Float` does not need a wrapper record for people to understand what it means,
so could you please change the type to

```
type alias ProgressIndicator =
    Maybe Float
```